src/ValueObjects/CalendarSummary.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\SearchV3\ValueObjects;
+
+use InvalidArgumentException;
+
+final class CalendarSummary
+{
+    private const LANGUAGES = [
+        'nl',
+        'fr',
+        'de',
+        'en',
+    ];
+
+    private const TYPES = [
+        'text',
+        'html',
+    ];
+
+    private const FORMATS = [
+        'xs',
+        'sm',
+        'md',
+        'lg',
+    ];
+
+    /**
+     * Associative array in the form of [language][type][format]
+     * @var array
```

Comment:
```suggestion
     * @var array<string,array<string,array<string,string>>>
```

Reply:
A format like this can be validated by tools like `phpstan` or `psalm` and should be understood by PHPStorm